app/views/users/index.html.erb

```diff
@@ -1 +1,20 @@
-<h1>Here is a list of users<h1/>
\ No newline at end of file
+<section class="my-blog bg-secondary bg-gradient">
+    <div class="container">
+		<!-- section title -->
+		<div class="row justify-content-md-center">
+			<div class="col-xl-5 col-lg-6 col-md-8">
+				<div class="section-title text-center title-ex1">
+					<h2>All User Posts</h2>
+					<p>Click the User to view Posts</p>
+				</div>
+			</div>
+		</div>
+		<!-- section title ends -->
+		<div class="row">
+			<% @users.each do |user| %>
+
+            	<%= render partial: 'user', locals: { user: user } %>
+			<% end %>
+		</div>    
+	</div>
```

Comment:
- [ ] You are making use of a partial here but the partial file is missing and it's causing the app to crash as a result of not locating the partial template being passed here. Kindly fix that in this file and the others 👍🏽

<img width="1158" alt="Screenshot 2023-11-03 at 10 30 13 PM" src="https://github.com/maths29/iwuBlog/assets/50941074/a4c15f28-37dd-429d-b96c-da71d6c112da">